core/controller/cluster/controller_basic_v2.go

```diff
@@ -139,7 +139,8 @@ func (c *controller) CreateClusterV2(ctx context.Context, applicationID uint, en
 	}
 
 	// 7. get templateRelease
-	tr, err := c.templateReleaseMgr.GetByTemplateNameAndRelease(ctx, r.TemplateInfo.Name, r.TemplateInfo.Release)
+	tr, err := c.templateReleaseMgr.GetByTemplateNameAndRelease(ctx,
+		buildTemplateInfo.TemplateInfo.Name, buildTemplateInfo.TemplateInfo.Release)
```

Comment:
Why use buildTemplateInfo instead of c.TemplateInfo

Reply:
buildTemplateInfo is generate from app config and request